aggs-injectable_test.go

```diff
@@ -0,0 +1,156 @@
+package aggretastic
+
+import (
+	"errors"
+	"github.com/olivere/elastic"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"reflect"
+)
+
+func Call(function interface{}, params ...interface{}) (result []reflect.Value, err error) {
+	f := reflect.ValueOf(function)
+	if len(params) != f.Type().NumIn() {
+		err = errors.New("The number of params is not adapted.")
+		return
+	}
+	in := make([]reflect.Value, len(params))
+	for k, param := range params {
+		in[k] = reflect.ValueOf(param)
+	}
+	result = f.Call(in)
+	return
+}
+
+var _ = Describe("Injectable", func() {
+	ch1 := make(chan Aggregation, len(aggMap["Injectable"]))
```

Comment:
this test file is for testing `aggs-injectable.go`. No, reason to have `aggMap` being involved here. 
`aggs-injectable` can be tested even if no real aggregations (from olivere) are here. Injectable strut has its own methods to be tested